### ISSUE TITLE
`no-multiple-resolved` wrongly reports `reject` in try/catch block

### ISSUE THREAD

**Issue by burtek** [eslint-community/eslint-plugin-promise#443]
From #382, which I think wasn't fully fixed

```js
try {
    resolve(someThrowingFn())
} catch (error) {
    reject(error)
}
```

still errors for me. It doesn't error if I do this:

```js
try {
    const data = someThrowingFn()
    resolve(data)
} catch (error) {
    reject(error)
}
```

But those two are identical and should behave same way?

**Comment by brettz9** [comment]
This issue has apparently already been resolved as I am not able to replicate. Closing though feel free to comment further as needed.
